packages/filefeeds/test/index.html

```
@@ -22,13 +22,13 @@
     <header style="padding: 0 1em">
       <h1>OneSchema FileFeeds JS SDK Test</h1>
 
-      <p style="margin: 0.5em 0"">
+      <p style="margin: 0.5em 0">
         <button id="launch-button">Launch</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)